src/main/java/com/example/booksstore/constraints/FieldMatch.java

```diff
@@ -11,6 +11,8 @@
 @Target({ElementType.TYPE})
 @Retention(RetentionPolicy.RUNTIME)
 public @interface FieldMatch {
+    String first();
+    String second();
     String message() default "Password and repeated password are not the same.";
```

Comment:
default message should not mention any fields because it can be used anywhere, so let's move this message to the user request to where we use it already and have the default here say something like `passed fields do not match`